lib/widgets/generate_compatible_xprv.dart

```diff
@@ -140,12 +141,12 @@ class GenerateCompatibleXprvState extends State<GenerateCompatibleXprv>
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
               Text(
-                'Password',
+                AppLocalizations.of(context)!.passwordLabel,
                 style: theme.textTheme.titleSmall,
               ),
               SizedBox(height: 8),
               InputLogin(
-                hint: 'Password',
+                hint: AppLocalizations.of(context)!.passwordLabel,
```

Comment:
Add localization to 'Confirm password' label in the label and input hint down below.